src/Layer/ColorLayer.js

```diff
@@ -84,7 +83,7 @@ class ColorLayer extends RasterLayer {
         deprecatedColorLayerOptions(config);
         super(id, config);
         this.isColorLayer = true;
-        this.style = new Style(config.style);
+        this.style = config.style;
```

Comment:
the line could be removed?

Reply:
Layer doesn't have a style properties.
In this PR we added the fact that if there is a style properties in options at Layer we change it to a new Style(config.style).